perception/traffic_light_classifier/src/cnn_classifier.cpp

```diff
@@ -39,10 +39,13 @@ CNNClassifier::CNNClassifier(rclcpp::Node * node_ptr) : node_ptr_(node_ptr)
   input_c_ = node_ptr_->declare_parameter("input_c", 3);
   input_h_ = node_ptr_->declare_parameter("input_h", 224);
   input_w_ = node_ptr_->declare_parameter("input_w", 224);
+  auto input_name = node_ptr_->declare_parameter("input_name", "input_0");
+  auto output_name = node_ptr_->declare_parameter("output_name", "output_0");
+  apply_softmax_ = node_ptr_->declare_parameter("apply_softmax", false);
```

Comment:
Would it be helpful to also declare std and mean here? for people who use their custom-trained models

like this Pr : https://github.com/autowarefoundation/autoware.universe/pull/2794

Reply:
Yes, I think so. We can merge the PR after merging this PR.

Reply:
Also, ı would like to add, really loved this single inference debug tool, really useful for users, thanks for it  :)